utils/data_write_options/data_write_options.py

```diff
@@ -194,5 +196,27 @@ def intf_antenna_count(self):
 
         return self._intf_antenna_count
 
+    @property
+    def main_antennas(self):
+        """
+        Gets the index of antennas in the main array corresponding to the receiver channels.
```

Comment:
transceiver instead of receiver for the main array antennas?

Reply:
Seems reasonable